golem-cli/Cargo.toml.nix

```diff
@@ -0,0 +1,82 @@
+[package]
```

Comment:
Why is there a copy of our `golem-cli` cargo file here? Who and when will keep it updated? What is the difference?

Reply:
the difference between this file and Cargo.toml is the change to of golem examples to be to one that is brought by nix (inputs.golem-examples, and additional missing reference), 
I think I had planned to be included in the update-deps command on the commands.nix but i might missed something I could look at it this weekend.